snowshu/adapters/target_adapters/postgres_adapter/postgres_adapter.py

```diff
@@ -15,7 +15,7 @@
 
 class PostgresAdapter(BaseTargetAdapter):
     name = 'postgres'
-    dialect = 'postgres'
+    dialect = 'postgresql'
```

Comment:
This fixes warning:
```
tests/integration/snowflake/test_analyze.py::test_analyze_unsampled
  /usr/local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py:263: SADeprecationWarning: The 'postgres' dialect name has been renamed to 'postgresql'
    loader = self.auto_fn(name)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

Reply:
Yeah, probably worth pulling the trigger on this change. Originally I was always waiting for the point that the pins would be updated to make this change, but that day never seemed to arrive.